### DISCUSSION TITLE
Collapse and expand built in options in Wiki.js

### DISCUSSION THREAD

**Discussion by rk408** [requarks/wiki#6792] [Help / Questions]
How to create collapsible and expandable content in wiki.js ?
Are there any built in direct options to do that or how to do it? Appreciate any support in this. 

Thanks,
Siva.

**Comment by P4mes4n** [comment]
The HTML Spoiler Tag allows for collapsible content. And the Tabsets as well, to some extend.
Maybe take a look [here](https://github.com/requarks/wiki/discussions/5497)
